docs/content/querying/groupbyquery.md

```diff
@@ -214,3 +214,4 @@ When using the "v2" strategy, the following query context parameters apply:
 |`maxMergingDictionarySize`|Can be used to lower the value of `druid.query.groupBy.maxMergingDictionarySize` for this query.|
 |`maxOnDiskStorage`|Can be used to lower the value of `druid.query.groupBy.maxOnDiskStorage` for this query.|
 |`sortByDimsFirst`|Sort the results first by dimension values and then by timestamp.|
+|`pushDownLimit`|Limits/orderbys are applied at the merge buffer level (only the top N results are kept in the merge buffers, reducing the workload at the broker level), when all fields in the orderby are part of the grouping key. When the sorting order uses fields that are not in the grouping key, this can result in approximate results with unknown accuracy, so this optimization is disabled by default in that case. Enabling this context flag turns on limit push down for limit/orderbys with non-grouping key columns.|
```

Comment:
@himanshug changed docs to not mention the merge buffers

@nishantmonu51 If the fields in the order by are all part of the grouping key but pushDownLimit is explicitly set to false, the the limit push down will still happen.

Renamed this to forcePushDownLimit